src/openrct2-ui/windows/LoadSave.cpp

```diff
@@ -380,7 +376,8 @@ namespace OpenRCT2::Ui::Windows
 
         void OnResize() override
         {
-            WindowSetResize(*this, kWindowSizeMin.width, kWindowSizeMin.height, kWindowSizeMax.width, kWindowSizeMax.height);
+            WindowSetResize(
+                *this, { kWindowSizeMin.width, kWindowSizeMin.height }, { kWindowSizeMax.width, kWindowSizeMax.height });
```

Comment:
Seems you can simplify this by just passing `kWindowSizeMin` and `kWindowSizeMax` directly?

Reply:
Absolutely, nice catch! (Clearly, I just used a regex to replace all these.)